### PR TITLE
Fledging - lesson 2

### PR DESCRIPTION
**Pull request by orta**
It's currently my overview, will flesh out with wording etc.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Ace, :shipit:

**Comment by orta** [comment]
OK this is ready to go

File: Fledgling/Lesson Two/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+So, last time we talked a little bit about non-trivial Objects (this wasn't noted, we looked into the [NSDate](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSDate_Class/index.html) class) These objects are made up from the types that we've been playing with: `Int`s, `String`s, `Array`s, `Bool`s. 
+
+We call the Types that are included inside Swift, the "Standard Library." These are the effectively building blocks of all other Types. However, it's hard to create an application with just numbers and strings. So, we bundle collections of related objects together into libraries or frameworks.
+
+Apple provides a set of libraries for you to use, in this case, there is one called `Foundation`. `Foundation` provides `NSDate` for dates, `NSFileManager` for getting information in/out of the file system, `NSTask` for running command line tasks, `NSURL` for representing a URL and a lot more. These are all considered foundational tools. 
+
+Separate from `Foundation` is `Cocoa` or `UIKit` - these provide all of the user interface objects, which we may touch on later in the week.
```
Comment (ashfurrow) [inline Fledgling/Lesson Two/README.md:49]: Later in the course?

**Comment by ashfurrow** [comment]
No blockers 💯

**Pull request merged**